### ISSUE TITLE
[docs] Missing docs for Storage.from_buffer

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#30387]
https://pytorch.org/docs/master/storage.html#torch.FloatStorage.from_buffer

**Comment by albanD** [comment]
Not a good justification but I think the historical reason is that we did not want the users to use it directly as it is fairly unsafe :)

**Comment by vadimkantorov** [comment]
In absence of reinterpret functionality, this can serve as a crude substitute... In my case I read some external program byte output. I think this scenario is fairly frequent

**Comment by vadimkantorov** [comment]
But if there is another method for this use case and reinterpret, I would be happy to abandon this from_buffer thing :)

**Comment by albanD** [comment]
I agree that a good documentation explaining the details of how to use this function and a warning that is it only advanced user would be useful.

**Comment by tandelDipak** [comment]
I want to reverse ToTensor transformation, Can someone add documentation for from_buffer().
